static/app/views/replays/deadClickList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as Layout from 'sentry/components/layouts/thirds';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function DeadClickList() {
+  const organization = useOrganization();
+  const hasDeadCicks = organization.features.includes(
+    'session-replay-rage-dead-selectors'
+  );
+
+  return hasDeadCicks ? (
+    <SentryDocumentTitle title={`Top Selectors with Dead Clicks — ${organization.slug}`}>
```

Comment:
nit: translation needed, and i think the org.slug part will be automatically added... there might be a separate prop `organization={organization}` though, i'm not 100% sure and need to look.

Reply:
yup you're right, there's a prop for `orgSlug`